### PR TITLE
Sell land 

### PR DESCRIPTION
**Pull request by klikooo**
The agent can now sell land when he is low on money. 

He first tries to sell land of category nature and after that of category garden.
The prices per m2 could also be a little bit tweaked to more realistic ones,

Please merge #24 first

### CODE REVIEW

**Comment by klikooo** [comment]
FIxed conflicts, you can test it now

File: src/DUWOagent/noBudgetMod.mod2g [hunk 1]
```diff
@@ -7,23 +7,22 @@ order=linear.
...
 
+define sell_building(Category, BuyerID, MultiPolygon)
+	as bel(not(sell_proposal(_,_)) % We are not already selling something
+		, getAreaOwnBuilding(_, Category, MultiPolygon, Area), Area > 10
+		, stakeholder(BuyerID, _)
+		, not(self(BuyerID)) % We cant sell to ourself.
+		, not(sell_denied(BuyerID, MultiPolygon))
+		, BuyerID \= 1). %Exclude ID 1 since this one is not used
+	
+
```
Comment (MarcianoJ) [inline src/DUWOagent/noBudgetMod.mod2g:18]: Place commas at the end of the line, like other files

**Comment by Denpeer** [comment]
travis error, probably something wrong with tygron. Should be fixed when you rerun it.

File: src/DUWOagent/requestEvents.mod2g [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+	
+	% A sell proposal has been denied
+	if bel(sell_proposal(BuyerID, MultiPolygonX), self(MyID)) ,
+		percept(requests(AllRequests)),
+		bel(member(request('INTERACTION','SELL_LAND',_,_,[MyID], _ , Price , 
+			MultiPolygon, _, [answer(0,'OK')]), AllRequests),
+			Price = 'NO_PRICE')
+	then delete(sell_proposal(BuyerID, MultiPolygonX)) + insert(sell_denied(BuyerID, MultiPolygonX)).
+
+}
```
Comment (MarcianoJ) [inline src/DUWOagent/requestEvents.mod2g:24]: Fix indentation

**Comment by klikooo** [comment]
@Denpeer starting the travis build did indeed make it pass.

File: src/DUWOagent/noBudgetMod.mod2g (hunk 1 above)
Comment (klikooo) [inline src/DUWOagent/noBudgetMod.mod2g:18]: We will place commas at the beginning of a line in another pull requests

**Comment by Denpeer** [comment]
the agent seems stuck in lowbudgetmod when i run it

**Comment by MaxGroenenboom** [comment]
I cannot test it, the client crashes systematically for me now. Looking at the log everything that works still works though.

**Comment by MaxGroenenboom** [comment]
My manual test works, travis fails though, but I suppose that's a random failure. I approve as soon as travis builds again.

**Comment by MaxGroenenboom** [comment]
Alright I approve.

**Comment by Denpeer** [comment]
It works right up till the point when the tygron server crashes, so i approve

**Comment by MarcianoJ** [comment]
I second @Denpeer, approved

**Pull request merged**